Assets/Scripts/UI/Canvas/ColorPicker/ColorPicker.cs

```diff
@@ -0,0 +1,70 @@
+using UnityEngine;
+
+using Framework.Enums;
+
+namespace UI.Canvas.ColorPicker
+{
+    public class ColorPicker : MonoBehaviour
+    {
+        [SerializeField] private LineRenderer routeLineRenderer;
+    
+        private void Start()
+        {
+            gameObject.SetActive(false);
+        }
+
+        /// <summary>
+        /// Checks which RouteColors enum goes with which color
+        /// </summary>
+        /// <param name="lineColor"> The RouteColors enum that gets grabbed from the button that's clicked </param>
+        /// <returns> Returns the color that is used for the ChangeColor function </returns>
+        public Color SetColor(RouteColors lineColor)
+        {
+            Color newColor = new Color();
+            
+            switch (lineColor)
+            {
+                case RouteColors.RED:
+                    newColor = new Color(1f,0f,0f);
+                    break;
+                case RouteColors.GREEN:
+                    newColor = new Color(0f, 1f, 0f);
+                    break;
+                case RouteColors.BLUE:
+                    newColor = new Color(0f, 0f, 1f);
+                    break;
+                case RouteColors.YELLOW:
+                    newColor = new Color(1f, 1f, 0f);
+                    break;
+                case RouteColors.MAGENTA:
+                    newColor = new Color(1f, 0f, 1f);
+                    break;
+                case RouteColors.ORANGE:
+                    newColor = new Color(1f, 0.5f, 0f);
+                    break;
+            }
+            
+            return newColor;
+        }
+        
+        /// <summary>
+        /// Changes the line renderer's line color to a new color
+        /// </summary>
+        /// <param name="newColor"> The color to set the line to </param>
+        public void ChangeColor(ColorChange colorChange)
+        {
+            routeLineRenderer.startColor = SetColor(colorChange.ColorToChange);
+            routeLineRenderer.endColor = SetColor(colorChange.ColorToChange);
+        }
+
+        /// <summary>
+        /// Changes the line renderer's line width to a new width
+        /// </summary>
+        /// <param name="newWidth"> The width to set the line to </param>
+        public void ChangeWidth(float newWidth)
+        {
+            routeLineRenderer.startWidth = newWidth;
+            routeLineRenderer.endWidth = newWidth;
+        }
```

Comment:
```suggestion
```

Ik ben er niet mee eens dat een `ColorPicker` class een `ChangeWidth` functie heeft, dit is geen SRP. Maak hiervoor een nieuwe class aan op de route die dit doet.

Reply:
(changewidth werdt nog niet eens gebruikt)